microprot/scripts/snakemake_helpers.py

```diff
@@ -0,0 +1,91 @@
+import os
+import textwrap
+from datetime import datetime
+from microprot.scripts import process_fasta
+from skbio import io
+
+
+def not_empty(fname):
+    if os.path.exists(fname) and os.path.getsize(fname) > 0:
+        return True
+    else:
+        return False
+
+
+def trim(inp_str, symbol):
+    if isinstance(inp_str, str) and isinstance(symbol, str):
+        out = symbol.join(inp_str.split(symbol)[:-1])
+    else:
+        raise TypeError('Trim function requires strings as input!')
+    return out
+
+
+def msa_size(msa_fp):
+    msa_dir, msa_ext = os.path.splitext(os.path.abspath(msa_fp))
+    if msa_ext != '.a3m':
+        msa_ext = '.a3m'
+    with open(''.join([msa_dir, msa_ext]), 'r') as f:
+        lines = f.readlines()
+    msa_size = len([line for line in lines if line.startswith('>')])-1
```

Comment:
This may be an infficient way of using memory. How about `msa_size = sum(1 for line in lines if line.startswith('>')) - 1`.